apps/openmw/mwworld/worldimp.cpp

```diff
@@ -2941,9 +2934,20 @@ namespace MWWorld
         return mScriptsEnabled;
     }
 
-    void World::loadContentFiles(const Files::Collections& fileCollections,
-        const std::vector<std::string>& content, const std::vector<std::string>& groundcover, ContentLoader& contentLoader)
+    void World::loadContentFiles(const Files::Collections& fileCollections, const std::vector<std::string>& content, ESMStore& store, std::vector<ESM::ESMReader>& readers, ToUTF8::Utf8Encoder* encoder, Loading::Listener* listener)
```

Comment:
`contentLoader` is not passed as an argument now, so `contentLoader.load(col.getPath(file), idx);` below will not compile as well.